### DISCUSSION TITLE
How to `git rm` during a rebase for a file that was deleted from the base branch?

### DISCUSSION THREAD

**Discussion by massarom** [jesseduffield/lazygit#3777] [Q&A]
Sometimes it happens that I rebase a branch and there's a conflict where my branch has edited some file, but that file was recently deleted from the branch I'm rebasing onto. How do I "accept the deletion" of this file?

If press `Space`, then I stage my file as it in my branch. If I press discard, same thing happens, I restore "my version", which is not what I want. I always have to drop down to another terminal and do `git rm file` manually.

There's a similar discussion #2394, but it's not exactly the same. That sounds more generic to me, this is just in the context of fixing a rebase conflict. Don't know if it makes any difference.

**Comment by carlesba** [comment]
I have the same issue. Not sure how to keep the file deleted.
